packages/devtools_app/lib/src/screens/memory/memory_graph_model.dart

```diff
@@ -417,30 +422,30 @@ class HeapGraphElementLive extends HeapGraphElement {
   HeapGraphElementLive(this.origin);
 
   final HeapSnapshotObject origin;
-  HeapGraphClass theClass;
+  HeapGraphClass? theClass;
 
   @override
   bool get isSentinel => false;
 
-  HeapGraphElement getField(String name) {
+  HeapGraphElement? getField(String name) {
     if (theClass is HeapGraphClassLive) {
-      final HeapGraphClassLive c = theClass;
+      final HeapGraphClassLive c = theClass as HeapGraphClassLive;
       for (HeapSnapshotField field in c.origin.fields) {
         if (field.name == name) {
-          return references[field.index];
+          return references![field.index];
         }
       }
     }
     return null;
   }
 
-  List<MapEntry<String, HeapGraphElement>> getFields() {
-    final List<MapEntry<String, HeapGraphElement>> result = [];
+  List<MapEntry<String, HeapGraphElement?>> getFields() {
```

Comment:
can HeapGraphElement be non-null here too

Reply:
done